components/helper/ShareButtons.tsx

```diff
@@ -9,9 +12,131 @@ import {
   TwitterShareButton,
 } from 'react-share';
 
-export default function ShareButtons() {
+type ShareButtonsProps = {
+  popup: boolean;
+  setCopied: React.Dispatch<React.SetStateAction<boolean>>;
+  copied: boolean;
+};
+
+export default function ShareButtons({
+  popup, setCopied, copied,
+}: ShareButtonsProps) {
   const url = 'https://clearviction.org/calculator/head-initial-1-cont';
 
+  const style = {
+    fontSize: '1.25rem',
+    fontWeight: 400,
+    lineHeight: '1.375rem',
+    letterSpacing: '.0156rem',
+    textAlign: 'left',
+    color: '#2F3554',
+    textDecoration: 'underline',
+    marginLeft: '.4375rem',
+  };
+
+  const handleCopyClick = async () => {
```

Comment:
I had some unexpected behavior using the copy link button. I clicked from the main popup to copy link, and got the link copied popup, then I closed out. Then I clicked share the calculator button again and I got the link copied popup instead of the main popup.